compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/KeywordMap.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.compiler.internal.parser;
+
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class KeywordMap {
+
+    private static final Map<String, Keyword> keywordsMap = Stream.of(Keyword.values()).collect(Collectors
```

Comment:
Shall we add a method in the Keyword enum and get the corresponding enum to a given value there itself?

eg:
```
        Keyword getValue(string value) {
            keywordsMap.get(value);
        }
```